zotero_docai_pipeline/orchestration/pipeline.py

```
@@ -1921,18 +2213,48 @@ def run(self) -> dict[str, int | float | list[ProcessingResult]]:
+                                item["key"], self.zotero_config.tags.output
+                            )
+                            log_tagging(self.logger, self.zotero_config.tags.output)
+                            tag_adding_processed += 1
+                        except ZoteroClientError as e:
+                            self.logger.warning(
+                                f"Failed to add output tag to item {item['key']}: {e}"
+                            )
+                            item_tag_results[0].tags_failed.append(
+                                self.zotero_config.tags.output
+                            )
+                        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)